docs/plugins/npe2_migration_guide.md

```diff
@@ -0,0 +1,457 @@
+(npe2-migration-guide)=
+
+# npe2 migration guide
+
+We've introduced a new plugin engine. The new library [npe2][] is a
+re-imagining of how napari interacts with plugins. Rather than importing a
+package to discover plugin functionality, a static manifest file is used to
+declaratively describe a plugin's capabilities. Details can be found in the
+{ref}`npe2-manifest-spec`.
+
+Plugins targeting `napari-plugin-engine` will continue to work, but we
+recommend migrating to `npe2`. This guide will help you learn how!
+
+## Migrating using the `npe2` command line tool
+
+`npe2` provides a command line interface to help convert a
+`napari-plugin-engine`-style plugin.
+
+### 1. Install the `npe2` command
+
+```bash
+pip install npe2
+npe2 convert --help
+```
+
+The `npe2` tool provides a few commands to help you develop your plugin. In
+this case we're asking for help with the `convert` command:
+
+```
+Usage: npe2 convert [OPTIONS] PLUGIN_NAME
+
+  Convert existing plugin to new manifest.
+
+Arguments:
+  PLUGIN_NAME  The name of the plugin to convert  [required]
+
+Options:
+  --format [yaml|json|toml]  [default: yaml]
+  --out PATH
+  --help                     Show this message and exit.
+```
+
+### 2. Install your `napari-plugin-engine` based plugin.
+
+As an example, we'll walk through the process using the `napari-animation`
+plugin.
+
+We checkout the plugin and install it as a local editable package.
+
+```bash
+git clone https://github.com/napari/napari-animation.git
+cd napari-animation
+pip install -e .
+```
+
+### 3. Inspect package metadata
+
+The `napari-animation` package defines it's core metadata in `setup.cfg`.
+Inside, it defines an _entry point group_. That section should initially
+contain:
+
+```ini
+[options.entry_points]
+napari.plugin =
+    animation = napari_animation
+```
+
+That metadata gives the name of the plugin: "animation". The name is used in
+the next step. Later we're going to come back and update this section.
+
+### 4. Generate the plugin manifest
+
+Tse the npe2 command in the terminal:
```

Comment:
I don't know what `Tse` is supposed to be here but I suspect we need to fix this 😆

Reply:
```suggestion
To create the manifest, use the npe2 command in the terminal:
```

Reply:
😬